plugins/ledger/ledger.py

```diff
@@ -174,26 +175,26 @@ def perform_hw1_preflight(self):
             if self.has_detached_pin_support(self.dongleObject) and not self.is_pin_validated(self.dongleObject) and (self.handler is not None):
                 remaining_attempts = self.dongleObject.getVerifyPinRemainingAttempts()
                 if remaining_attempts != 1:
-                    msg = "Enter your Ledger PIN - remaining attempts : " + str(remaining_attempts)
+                    msg = _('Enter your {} PIN - remaining attempts: {}').format(self.device, remaining_attempts)
                 else:
-                    msg = "Enter your Ledger PIN - WARNING : LAST ATTEMPT. If the PIN is not correct, the dongle will be wiped."
+                    msg = _('Enter your {} PIN - WARNING: LAST ATTEMPT. If the PIN is not correct, the dongle will be wiped.').format(self.device)
```

Comment:
You missed a "dongle" here on like 180?  ;)

Reply:
Ah, thanks, will push an update.